paint/PAINT_DrawWindow.cpp

```diff
@@ -192,29 +204,24 @@ void DrawWindow::clearWindow() const
 //using Cohen-Sutherland clipping algorithm
 std::vector<win::Coords> DrawWindow::clippingHandler(win::Coords pStart, win::Coords pEnd) const
```

Comment:
Prefer this to be a utility function in gfx

```
namespace gfx {
    std::vector<Point> clipLineToRect(Rect const & r, Point const & start, Point const & end);
}
```
            
(Would need to move Coord from win -> gfx OR create a Point type in gfx and a function to convert (hopefully just a cast!))

Reply:
done, and moved Coord to gfx too